trax/data/tf_inputs.py

```diff
@@ -240,14 +240,19 @@ def TFDS(  # pylint: disable=invalid-name
     dataset_name,
     data_dir=None,
     keys=None,
+    tfds_preprocess_fn=None,
     train=True,
     eval_holdout_size=0):
-  """Returns an iterator of numpy arrays representing the dataset."""
+  """Returns an iterator of numpy arrays representing the dataset.
+  Args:
+     tfds_preprocess_fn: function that transforms tensorflow dataset applied before changing dataset to generator
```

Comment:
Could we either add the other args, or delete this arg and just leave the docstring?

Reply:
I added the other args in the docstring. It might be not clear what 'tfds_preprocess_fn' is just from the name, so I think it's better to include all args than none.